protocols/swap/test/Swap-unit.js

```diff
@@ -105,13 +105,15 @@ contract('Swap Unit Tests', async accounts => {
         signature,
       ]
 
-      //sender authorizes signer
-      emitted(
-        await swap.authorize(mockSigner, Jun_06_2017T00_00_00_UTC, {
+      //mockSender authorizes mockSigner using authorizeSender
+      //mocks that case where one can send unsigned orders if authorized
+      // the roles of this authorization need to be reversed for the swap to succeed
+      /*      emitted(
+        await swap.authorizeSender(mockSigner, Jun_06_2017T00_00_00_UTC, {
```

Comment:
Now that this is commented out, is there a reason you are not removing it entirely?